src/encoded/static/components/encyclopedia.js

```diff
@@ -57,130 +55,43 @@ const defaultAssemblyByOrganism = {
 };
 
 // Hide facets that we don't want to display
-const filterFacet = (response) => {
-    const newResponse = response;
-    const filteredFacets = response.facets.filter((facet) => (keepFacets.indexOf(facet.field) > -1));
-    newResponse.facets = filteredFacets;
-    return newResponse;
+const filterFacet = (facets) => {
+    const filteredFacets = facets.filter((facet) => (keepFacets.indexOf(facet.field) > -1));
```

Comment:
After the sprint is over, it would be cool to get your philosophical take on doing [].indexOf vs [].includes.